crud.py

```diff
@@ -0,0 +1,63 @@
+from sqlalchemy.orm import Session, Query
+
+import models
+import schemas
+
+
+def create_author(
+        db: Session,
+        author: schemas.AuthorCreate
+) -> models.DBAuthor:
+    db_author = models.DBAuthor(
+        name=author.name,
+        bio=author.bio
+    )
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+
+    return db_author
+
+
+def get_authors_list(
+        db: Session,
+        skip: int = 0,
+        limit: int = 100
+) -> Query:
+    return db.query(models.DBAuthor).offset(skip).limit(limit).all()
+
+
+def get_author(db: Session, author_id: int) -> Query:

```

Comment:
<img width="709" alt="image" src="https://user-images.githubusercontent.com/65166203/230024474-6de6e47a-615c-46c1-a435-9369de8a9129.png">
Is really return type Query?
Fix in all places